packages/near-membrane-base/src/membrane.ts

```diff
@@ -4212,82 +4233,15 @@ export function createMembraneMarshall(
                   }
                 : (noop as CallableInstallLazyPropertyDescriptors),
             // callableIsTargetLive
-            !IS_IN_SHADOW_REALM
-                ? (targetPointer: Pointer): boolean => {
+            !IS_IN_SHADOW_REALM && liveTargetCallback
```

Comment:
looking at this line, it looks like liveTarbgetCallback should we required. If it is not provided... then what?